docs/atlascli/command/atlas-streams-instances-describe.txt

```diff
@@ -0,0 +1,107 @@
+.. _atlas-streams-instances-describe:
+
+================================
+atlas streams instances describe
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Describe an Atlas Stream Processing instance for your project.
+
+To use this command, you must authenticate with a user account or an API key that has the Project Read Only role.
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas streams instances describe <name> [options]
+
+.. Code end marker, please don't delete this comment
+
+Arguments
+---------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - name
+     - string
+     - true
+     - Name of the Atlas Streams processor instance.
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -h, --help
+     - 
+     - false
+     - help for describe
+   * - -o, --output
+     - string
+     - false
+     - Output format. Valid values are json, json-path, go-template, or go-template-file. To see full output, use the -o json option.
+   * - --projectId
+     - string
+     - false
+     - Hexadecimal string that identifies the project to use. This option overrides the settings in the configuration file or environment variable.
+
+Inherited Options
+-----------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -P, --profile
+     - string
+     - false
+     - Name of the profile to use from your configuration file. To learn about profiles for the Atlas CLI, see https://dochub.mongodb.org/core/atlas-cli-save-connection-settings.
+
+Output
+------
+
+If the command succeeds, the CLI returns output similar to the following sample. Values in brackets represent your values.
+
+.. code-block::
+
+   ID     NAME     CLOUD                               REGION
+   <Id>   <Name>   <DataProcessRegion.CloudProvider>   <DataProcessRegion.Region>
+   
+
+Examples
+--------
+
+.. code-block::
+
+   # Return an Atlas Stream Processing instance with a specific name:
+   atlas streams describe myProcessor
+   
+.. code-block::
+
+   # Return a JSON-formatted Atlas Stream Processing instance with a specific name:
+   atlas streams describe myProcessor --output json
```

Comment:
Should this be `atlas streams instances describe myProcessor`?

Reply:
Resolved in [82c2186](https://github.com/mongodb/mongodb-atlas-cli/pull/2138/commits/82c2186c49ac36d4c34bfb1057ac3be1bb66941d)